extensions/jackson/deployment/src/main/java/io/quarkus/jackson/deployment/JacksonProcessor.java

```diff
@@ -100,6 +103,12 @@ public class JacksonProcessor {
     private static final String PARAMETER_NAMES_MODULE = "com.fasterxml.jackson.module.paramnames.ParameterNamesModule";
     private static final DotName JACKSON_MIXIN = DotName.createSimple(JacksonMixin.class.getName());
 
+    private static final ClassDesc OBJECT_MAPPER_CLASS_DESC = ClassDesc.of(ObjectMapper.class.getName());
+    private static final ClassMethodDesc OBJECT_MAPPER_REGISTER_MODULE_METHOD_DESC = ClassMethodDesc.of(
+            OBJECT_MAPPER_CLASS_DESC,
+            "registerModule",
+            MethodTypeDesc.of(OBJECT_MAPPER_CLASS_DESC, ClassDesc.of(Module.class.getName())));
+
```

Comment:
Since the class is statically known, you can do just `MethodDesc.of(ObjectMapper.class, "registerModule", ObjectMapper.class, Module.class)`.

Reply:
Oh, much nicer!